test/token/ERC721/extensions/ERC721Consecutive.t.sol

```diff
@@ -16,7 +16,10 @@ function toSingleton(address account) pure returns (address[] memory) {
 contract ERC721ConsecutiveTarget is StdUtils, ERC721Consecutive {
     uint256 public totalMinted = 0;
 
-    constructor(address[] memory receivers, uint256[] memory batches) ERC721("", "") {
+    constructor(address[] memory receivers, uint256[] memory batches, uint96 startingTokenId) ERC721("", "") {
+        if (startingTokenId > 0) {
+            _mintConsecutive(address(0), startingTokenId);
+        }
```

Comment:
Afaik, this is covered by "normal" batches. We just need to make sure that receiver = 0 is considered

Reply:
Although multiple tests use `vm.assume(receiver != address(0))`, they'll fail for `ownerOf` since [address zero is not a valid owner](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/eedca5d873a559140d79cc7ec674d0e28b2b6ebd/contracts/token/ERC721/ERC721.sol#L63), according to the ERC721 implementation.

I don't see an easy way around that, so the constructor if may be the easiest.